core/src/avm2/globals/flash/filters/ShaderFilter.as

```diff
@@ -16,10 +16,42 @@ package flash.filters {
             this._shader = value;
         }
 
-        public var bottomExtension:int = 0;
-        public var leftExtension:int = 0;
-        public var rightExtension:int = 0;
-        public var topExtension:int = 0;
+        private var _bottomExtension:int = 0;
+        private var _leftExtension:int = 0;
+        private var _rightExtension:int = 0;
+        private var _topExtension:int = 0;
+
+        public function get bottomExtension():int {
+            return this._bottomExtension;
+        }
+
+        public function set bottomExtension(value:int):void {
+            this._bottomExtension = value;
+        }
+
+        public function get leftExtension():int {
+            return this._leftExtension;
+        }
+
+        public function set leftExtension(value:int):void {
+            this._leftExtension = value;
+        }
+
+        public function get rightExtension():int {
+            return this._rightExtension;
+        }
+
+        public function set rightExtension(value:int):void {
+            this._ = value;
```

Comment:
typo here: _ --> _rightExtension
Also, the tests didn't catch this- are there no tests for the extensions currently?

Reply:
They're not hooked up to anything yet, so I haven't written any tests.